docs/triggerbindings.md

```diff
@@ -1,9 +1,8 @@
 # TriggerBindings
-As per the name, `TriggerBindings` bind events to `TriggerTemplates`.
-`TriggerBindings` build atop `TriggerTemplates` by enabling you to capture fields within an event payload and store them as parameters.
-These parameters are passed into the referenced `TriggerTemplate`.
-Although similar, the separation of these two resources was deliberate to encourage `TriggerTemplate` definitions to be reusable.
-Further, it is as this level that the service account is connected, which specifies what permissions the resources will be created (or at least attempted) with.
+As per the name, `TriggerBindings` bind against events/triggers.
+`TriggerBindings` enabling you to capture fields within an event payload and store them as parameters. 
+The separation of `TriggerBindings` from `TriggerTemplates` was deliberate to encourage reuse
+since a template can we well defined where the incoming data (whether event sources or data structure) might be variable.
```

Comment:
I think something is odd in the middle of this sentence, maybe "since a template can _be_ well defined"? 

i think the reason tho is really that you might be able to define one `TriggerBinding` that works for multiple `TriggerTemplates`?